q2_fmt/_peds.py

```diff
@@ -486,8 +507,58 @@ def _create_masking(time_metadata, donor_df, recip_df, reference_column):
 
 
 def _mask_recipient(donor_mask, recip_df):
-    recip_mask = donor_mask & recip_df
-    return recip_mask
+    maskedrecip = donor_mask & recip_df
+    return maskedrecip
+
+
+def sample_pprs(table: pd.DataFrame, metadata: qiime2.Metadata,
+                time_column: str, baseline_timepoint: str, subject_column: str,
+                filter_missing_references: bool = False,
+                drop_incomplete_subjects: bool = False,
+                drop_incomplete_timepoints: list = None) -> (pd.DataFrame):
+    ids_with_data = table.index
+    metadata = metadata.filter_ids(ids_to_keep=ids_with_data)
+    column_properties = metadata.columns
+    # TODO: Make incomplete samples possible move this to heatmap
+    metadata = metadata.to_dataframe()
+    if drop_incomplete_timepoints is not None:
```

Comment:
You can also just write this as `if drop_incomplete_timepoints:` - it's a bit more succinct that way, but also not a huge deal to leave as-is.